CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+set(PYTHON_SCRIPT_DIR python-scripts)
+macro(add_python_target target)
+    foreach (file ${target})
+        get_filename_component(file_name ${file} NAME)
+        configure_file(${file} ${PYTHON_SCRIPT_DIR}/${file_name} COPYONLY)
+    endforeach ()
+endmacro()
+
+cmake_minimum_required(VERSION 3.1.0)
+
+project(minotaur-cpp)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_PREFIX_PATH X:\\Qt\\5.8\\mingw53_32)
```

Comment:
this is platform specific, can you change this so that it works on all platforms?